mlir/test/Dialect/Vector/vector-transfer-collapse-inner-most-dims.mlir

```diff
@@ -217,16 +257,3 @@ func.func @leading_scalable_dimension_transfer_write(%dest : memref<24x1xf32>, %
 // CHECK:        %[[SUBVIEW:.+]] = memref.subview %[[DEST]][0, 0] [24, 1] [1, 1] : memref<24x1xf32> to memref<24xf32, strided<[1]>>
 // CHECK:        %[[CAST:.+]] = vector.shape_cast %[[VEC]] : vector<[4]x1xf32> to vector<[4]xf32>
 // CHECK:        vector.transfer_write %[[CAST]], %[[SUBVIEW]]{{.*}} {in_bounds = [true]} : vector<[4]xf32>, memref<24xf32, strided<[1]>>
-
-// -----
-
-// Negative test: [1] (scalable 1) is _not_ a unit dimension.
-func.func @trailing_scalable_one_dim_transfer_write(%dest : memref<24x1xf32>, %vec: vector<4x[1]xf32>, %index: index) {
-  %c0 = arith.constant 0 : index
-  vector.transfer_write %vec, %dest[%index, %c0] {in_bounds = [true, true]} : vector<4x[1]xf32>,  memref<24x1xf32>
-  return
-}
```

Comment:
There seems to be no replacement for this `transfer_write` negative test?

Reply:
> Also,
> 
> @leading_scalable_dimension_transfer_read,
> @trailing_scalable_one_dim_transfer_write,
> are replaced with:
> 
> @contiguous_inner_most_scalable_inner_dim,
> @negative_scalable_unit_dim,

Check `@negative_scalable_unit_dim` :) The idea was to make it a variation of `@negative_scalable_unit_dim` (so that it's easy to spot the difference).

Reply:
But `@negative_scalable_unit_dim` is checking `transfer_read`, while this test checks `transfer_write` (a different pattern).